_source/logzio_collections/_log-sources/trivy.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Ship reports from Trivy Operator
+logo:
+  logofile: trivy-logo.png
+  orientation: vertical
+data-source: Trivy Operator
+description: Ship reports from Trivy Operator
+data-for-product-source: Logs
+contributors:
+  - nshishkin
+shipping-tags:
+  - container
+order: 1380
+---
+[Trivy](https://www.aquasec.com/products/trivy/) is an open-source vulnerability scanner for containers that can detect vulnerabilities in OS packages and application dependencies. The [Trivy Operator](https://github.com/aquasecurity/trivy-operator) is a Kubernetes operator that automates the process of scanning container images for vulnerabilities using Trivy.
+
+This integration utilizes the logzio-monitoring Helm Chart to deploy the Trivy Operator within your Kubernetes cluster. The Trivy Operator scans container images for vulnerabilities and forwards the generated reports to Logz.io for further analysis and monitoring.
+
+<!-- info-box-start:info -->
+At present, only vulnerability reports are being collected.
+{:.info-box.note}
+<!-- info-box-end -->
+
+
+**Before you begin, you'll need**: 
+
+* an active account with Logz.io
+* Kubernetes cluster to send reports from
+
+<div class="tasklist">
+
+##### Select the namespace
```

Comment:
We can remove this step by adding a `--create-namespace` to the helm install command.